docker-compose.yml

```diff
@@ -25,12 +29,15 @@ services:
      S3_BUCKET_NAME: "${S3_BUCKET_NAME}"
      FLASK_APP: app
      FLASK_ENV: "${FLASK_ENV:-development}"
+     HMR: 1
    volumes:
      - ./OpenOversight/:/usr/src/app/OpenOversight/:z
+     - cache_dir:/usr/src/app/.cache
+     - /usr/src/app/OpenOversight/static/dist/
```

Comment:
I hope I merged the docker-compose conflicts correctly, but it would be good to check my work here